bindings/python/ailoy/models/api_model.py

```diff
@@ -44,7 +44,7 @@ class APIModel:
     provider: Optional[APIModelProvider] = None
 
     @model_validator(mode="after")
-    def validate_provider(self) -> Self:
+    def validate_provider(self):
```

Comment:
Would it be better to be `'APIModel'`?
```suggestion
    def validate_provider(self) -> 'APIModel':
```